app/google-services.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
I'm thinking that this contains many API keys which should not be public. Remove it from GIT, keep it ignored. Have it only locally when you are building it.